### ISSUE TITLE
Links in readme to Zeranoe now broken

### ISSUE THREAD

**Issue by voltagex** [rdp/ffmpeg-windows-build-helpers#530]
Zeranoe is no more, and appears to have completely pulled the forum - e.g. https://ffmpeg.zeranoe.com/forum/viewtopic.php?f=5&t=796&p=4095#p4095

Hope someone had a backup.

**Comment by nschlia** [comment]
Could be a temporary problem, according to whois the domain is still registered. But the DNS servers (operated by Cloudflare) don't seem to answer. dns.cloudflare.com seems to be authoritative:

```
~# dig zeranoe.com @dns.cloudflare.com.

; <<>> DiG 9.16.8-Debian <<>> zeranoe.com @dns.cloudflare.com.
;; global options: +cmd
;; connection timed out; no servers could be reached
```
Guess we need to be patient.

**Comment by Reino17** [comment]
No, it's not temporary. Zeranoe permanently stopped hosting his website and forum ± 4 months ago.

**Comment by nschlia** [comment]
> No, it's not temporary. Zeranoe permanently stopped hosting his website and forum ± 4 months ago.

Did not know. That's bad...

**Comment by rdp** [comment]
Maybe you can just "use" .lib files that it generates? https://stackoverflow.com/questions/11701635/use-ffmpeg-in-visual-studio/11701737
